magefile.go

```diff
@@ -444,27 +439,18 @@ func (Scaler) GenerateProto() error {
 
 // Create a new example HTTPScaledObject
 func NewHTTPSO(ctx context.Context, namespace string) error {
-	return sh.RunWithV(
-		make(map[string]string),
-		"kubectl", "create", "-f", "examples/httpscaledobject.yaml", "-n", namespace,
-	)
+	return sh.RunV("kubectl", "create", "-f", "./examples/v0.3.0/httpscaledobject.yaml", "-n", namespace)
```

Comment:
same here, can we make the path to the httpscaledobject configurable via an env var? can default to this